target_setting_runs/imagenet_resnet/jax_submission.py

```diff
@@ -26,15 +26,16 @@ def get_batch_size(workload_name):
 @functools.partial(
     jax.pmap,
     axis_name='batch',
-    in_axes=(None, None, 0, 0, 0, 0, 0),
-    static_broadcasted_argnums=(0, 1))
+    in_axes=(None, None, 0, 0, 0, 0, 0, None),
+    static_broadcasted_argnums=(0, 1, 7))
```

Comment:
(same with the other submission.py files) do we need to specify label_smoothing as a static arg? I doubt it matters, but in the rare case it would ever change during training it would trigger a jax recompilation

Reply:
I also briefly considered that but at least for the target-setting runs label smoothing is never changing during training or is it? And people can obviously do whatever they want in their own submissions.

Reply:
Ok, I will just not specify it as static, probably doesn't hurt anyway.